R/row_to_names.R

```diff
@@ -139,3 +146,9 @@ find_header <- function(dat, ...) {
   }
   ret
 }
+
+reduce_na <- function(x) {
```

Comment:
If the above suggestion to use `sapply` is correct, then this function is no longer needed.

Reply:
It was removed.

Finally, I wanted to note that when using `remove_rows_above = TRUE`, I opted to start from the first row of the input in `row_number`.
Another idea would be to default to the topmost row, for example when the input is something like `row_number = 2:1`, but this is your decision to make, I guess.